jac/jaclang/compiler/parser.py

```diff
@@ -215,7 +215,11 @@ def global_var(self, kid: list[ast.AstNode]) -> ast.GlobalVars:
             global_var: (KW_LET | KW_GLOBAL) access_tag? assignment_list SEMI
             """
             is_frozen = isinstance(kid[0], ast.Token) and kid[0].name == Tok.KW_LET
-            access = kid[1] if isinstance(kid[1], ast.SubTag) else None
+            access = (
```

Comment:
NOTE: SubTag will be removed  and the token will be handled properly, for now I'm doing a type ignore since SubTag's tag is a generic type (Not token).